src/util/Iguana.rsc

```diff
@@ -3,15 +3,16 @@ module util::Iguana
 extend ParseTree;
 import IO;
 
-syntax A = "a";
+// import lang::rascal::\syntax::Rascal;
+import demo::lang::Pico::Syntax;
 
 alias Parser[&T] = &T (str input, loc origin);
 
 @javaClass{util.ParserGenerator}
-java Parser[&T] parser(type[&T] grammar); 
+java Parser[&T] createParser(type[&T] grammar);
```

Comment:
I had to rename this method, somehow there was a conflict and Rascal was trying to load the default parse method.